etc/defaults.virtual

```diff
@@ -36,3 +36,4 @@ xserver-abi-video xorg-server
 libGL libglvnd
 libEGL libglvnd
 libGLES libglvnd
+oci-runtime runc
```

Comment:
Virtual packages are sort of seriously broken. If they can be avoided, that would be preferable.